pypeit/scripts/chk_calibs.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script examines a set of files and indicates which do and
+which do not have sufficient calibs
+"""
+import argparse
+from pypeit import defs
+
+from IPython import embed
+
+def parser(options=None):
+    # TODO: Add argument that specifies the log file
+    parser = argparse.ArgumentParser(description="Script to setup a PypeIt run [v3]")
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-r', '--root', type=str, default=None,
+                       help='File path+root, e.g. /data/Kast/b ')
+    parser.add_argument('-s', '--spectrograph', default=None, type=str,
+                        help='A valid spectrograph identifier: {0}'.format(
+                                ', '.join(defs.pypeit_spectrographs)))
+    parser.add_argument('-e', '--extension', default='.fits',
+                        help='File extension; compression indicators (e.g. .gz) not required.')
+
+    return parser.parse_args() if options is None else parser.parse_args(options)
+
+
+def main(args):
+    """
+
+    Args:
+        args:
+
+    Returns:
+        astropy.table.Table:
```

Comment:
Normally `main` methods only return an exit status integer.

Reply:
Ah.  Got it.  You use it in the test.  Makes sense, but for your consideration: You can also catch output strings and test that the correct ones are printed.  See line 486 of test_datacontainer.py for the incantation.  Not sure this will work for this application, but thought I'd bring it up.

Reply:
roger that.